test/bdd/webhook/main.go

```diff
@@ -30,6 +34,17 @@ func connections(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		w.WriteHeader(http.StatusBadRequest)
 	}
+
+	connMsg := didexchange.ConnectionMsg{}
+	err = json.Unmarshal(msg, &connMsg)
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+	}
+	// check for action event and log the connectionID
+	if connMsg.State == "null" {
```

Comment:
we send null on the web hook? That seems like it should be reviewed.

Reply:
Created an issue - https://github.com/hyperledger/aries-framework-go/issues/690.